WcaOnRails/app/jobs/compute_linkings.rb

```diff
@@ -5,7 +5,7 @@ class ComputeLinkings < ApplicationJob
 
   def perform
     last_computation = Timestamp.find_or_create_by(name: 'linkings_computation')
-    if last_computation.date.nil? || last_computation < 3.days.ago
+    if last_computation.date.nil? || last_computation.date < 3.days.ago
```

Comment:
Sorry about this, but when I tested it on the staging the first condition must've passed, so that I didn't get the error. That said, the code should never look like this.